group_vars/all.yml.sample

```diff
@@ -114,6 +114,10 @@ dummy:
 #ceph_stable_release: dummy
 #ceph_stable_repo: "{{ ceph_mirror }}/debian-{{ ceph_stable_release }}"
 
+#nfs_ganesha_stable: true # use stable repos for nfs-ganesha
```

Comment:
@leseb is this file, along with group_vars/rhcs.yml.sample the correct place to have these variables? At first I had put them in roles/ceph-nfs/defaults/main.yml since the changes only matter on nfs nodes but after I saw how ceph_dev and ceph_stable are in this file and followed suite.